pb-crm-ui/src/app/core/dialog/dialog.service.ts

```diff
@@ -0,0 +1,30 @@
+import { Injectable } from '@angular/core';
+import {MatDialog, MatDialogRef} from '@angular/material/dialog';
+import {DialogInterface } from './temp-dialog.interface';
+import { DialogComponent } from './dialog.component';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class DialogService {
```

Comment:
This name needs to be more descriptive. We have a bunch of dialogs at this point and this only acts on one